lib/index.css

```diff
@@ -15,6 +15,12 @@
   display: block !important;
 }
 
+#telescope hr {
+  border: 1px solid var(--telescope-text-color);
+  margin: 1em 0;
+  display: block !important;
```

Comment:
hmmm id prefer not to use !important here if people want to override. Do you need this for it to apply? And if so, can you try `#telescope hr hr` to see if that makes it more specific to override whatever other rule is taking it?